dcat/index.html

```diff
@@ -3290,6 +3290,74 @@ <h2>Versioning</h2>
+</ol>
+
+<p>These version types are not necessarily mutually exclusive, and they may overlap - especially, the first and the second ones. So, the relationship(s) used to link resource versions depends on which aspect is relevant for the specific use case. E.g.:</p>
+<ul>
+<li>In same cases, the only relevant information is that resource <var>A</var> is a version of (<code>dct:isVersionOf</code> / <code>dct:hasVersion</code>) resource <var>B</var>, irrespective of the version type.</li>
+<li>In other cases, it is important to explicitly state that resource <var>A</var> replaces (<code>dct:replaces</code> / <code>dct:replacedBy</code>) resource <var>B</var>, and that it is the result of a revision (<code>prov:wasRevisionOf</code> / <code>prov:hadRevision</code>).</li>
+<li>Other cases may require having an explicit indication about which is the previous, next, and last version of a resource (<code>adms:prev</code>, <code>adms:next</code>, <code>adms:last</code>).</li>
+</ul>
+
+<p>The third version type covers more heterogeneous cases, compared with the other version types. Depending on the application scenario, some or all of them may not be considered strictly related to versioning, but rather completely different resources, derived from other ones. E.g., the transformation of a distribution in additional formats (from CSV to XML and JSON), units of measurement, or coordinate reference systems, may be simply considered as alternative distributions associated with the same dataset. The same could be said for translations from one natural language to another one.</p>
+
+<p>These cases can be addressed by using properties as <code>dct:source</code> and <code>prov:wasDerivedFrom</code> / <code>prov:hadDerivation</code>, as illustrated in <a href="#examples-dataset-provenance"></a> and <a href="#qualified-relationship"></a>. Moreover, for translations in natural language, property <a data-cite="VOCAB-ADMS#adms-translation"><code>adms:translation</code></a> could be used.</p>
+
+</section>
+
+<section id="version-info">
+<h2>Version information</h2>
+
+<p>Besides the relationships illustrated in the previous section, versioned resources may be associated with additional information, describing, e.g., their differences with the original resource (the version "delta").</p>
+
+<p>Two typical examples are (a) a description (textual or machine-actionable) of the changes between the current and the previous version of a resource, and (b) whether the new version of the resource is backward compatible or not with the previous version (this is particularly relevant when the resource is used by given applications, so it is fundamental to know if the new version can be also used with them).</p>
+
```

Comment:
I would add an editorial note to warn about the use of owl:backwardCompatibleWith/owl:incompatibleWith and open a specific issue to ease the discussion on this topic. Does this make sense for you (@andrea-perego)?

```suggestion

<aside class="ednote">
<p>Section 3.5 in  [OWL 2 specification](https://www.w3.org/TR/2012/REC-owl2-syntax-20121211/#Ontology_Annotations)  suggests  <code>owl:backwardCompatibleWith</code> / <code>owl:incompatibleWith</code> properties on entities other than ontologies is discouraged. Should we consider any other valuable alternatives? (see issue XXX for discussion) </p>
</aside>

```